src/ngraph/op/fused/group_conv_transpose.cpp

```diff
@@ -52,6 +52,7 @@ op::GroupConvolutionTranspose::GroupConvolutionTranspose(const shared_ptr<Node>&
 
 op::GroupConvolutionTranspose::GroupConvolutionTranspose(const std::shared_ptr<Node>& data,
                                                          const std::shared_ptr<Node>& filters,
+                                                         const PadType& pad_type,
```

Comment:
This changes an API signature. Can we retain the old constructor for two releases?